tests/models/test_responses.py

```diff
@@ -31,7 +30,6 @@ def test_response():
     assert response.text == "Hello, world!"
     assert response.request.method == "GET"
     assert response.request.url == "https://example.org"
-    assert response.elapsed >= datetime.timedelta(0)
```

Comment:
Haven't checked, do we still have some tests for the `response.elapsed` property?

Reply:
Yup `test_client.py` and `test_async_client.py` check the property, and `test_responses.py` has a test to ensure that it's not accessible until the response has been closed.